### DISCUSSION TITLE
Any way to name a cell without rerunning it?

### DISCUSSION THREAD

**Discussion by louy2** [fonsp/Pluto.jl#1928] [Questions about using Pluto]
I often run a one shot cell, and discover that I need to use it later, and I give it a name but it needs to be run again. Is there a way to name a cell without rerunning? Or is there a default name for the cell output?

**Comment by icweaver** [comment]
Just linking the related discussion here for future peeps

https://julialang.zulipchat.com/#narrow/stream/243342-pluto.2Ejl/topic/Rename.20cell.20without.20rerun.20or.20default.20cell.20name

**Comment by lungben** [comment]
Thinking it a bit further:

Currently every Pluto cell has an (internal) UUID, which is used internally and in the notebook.jl file, but is not exposed in the frontend.

We could show a unique identifier for each cell in the frontend, but this should be something nicer than the UUID. Some ideas:

* a number, maybe related to cell execution order (analogue to Jupyter)
* a short form of the UUID
* some nice symbol

This cell identifier could then be used for a number of different purposes:

* Use the identifier in other cells to access the cell return value (like in Jupyter)
* For visualization of cell dependencies or execution order
